### PR TITLE
Add IPv6 support for expose tests

### PR DESCRIPTION
**Pull request by ardaguclu**

### CODE REVIEW

File: test/extended/testdata/bindata.go [hunk 1]
```diff
@@ -34135,6 +34137,42 @@ func testExtendedTestdataCmdTestCmdTestdataApplicationTemplateStibuildJson() (*a
...
+var _testExtendedTestdataCmdTestCmdTestdataExternalServiceIpv6Yaml = []byte(`apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    external: service
+  name: external
+  resourceVersion: "1"
+  uid: 19cff995-5546-11e5-9f57-080027c5bfa9
+spec:
+  clusterIP: fd02::1
```
Comment (stbenjam) [inline test/extended/testdata/bindata.go:34149]: fd02::1 is already taken, I'd pick something else a bit higher.

**Comment by stbenjam** [comment]
@ardaguclu Please also fix the title

File: test/extended/testdata/bindata.go (hunk 1 above)
Comment (ardaguclu) [reply test/extended/testdata/bindata.go:34149]: fd02::8 and fd02::9 should work.

**Comment by ardaguclu** [comment]
@deejross Since https://github.com/openshift/origin/pull/26597 is merged. I'm closing this PR. You can open new unrevert PR as @deads2k suggested.

/close

**Pull request closed without merging**